tests/fixtures/browser.fixture.ts

```
@@ -0,0 +1,70 @@
+ * Only extend fixtures if CHROME_DEBUG_PORT is set
+ * Otherwise, use the base test which has default Playwright behavior
+ */
+const debugPort = process.env.CHROME_DEBUG_PORT;
+
+export const test = debugPort
+  ? base.extend<{
+      browser: Browser;
+      context: BrowserContext;
+      page: Page;
+    }>({
+      browser: async ({ }, use) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)